web/src/components/Callout.tsx

```diff
@@ -39,35 +39,59 @@ export const calloutComponentMapping: Record<string, CalloutMappingObj> = {
   },
 };
 
-export const Callout = (props: Props): ReactElement => {
+export const Callout = (props: PropsWithChildren<Props>): ReactNode => {
   const { Config } = useConfig();
+  const showIcon = Boolean(props.showIcon ?? false);
+  const showHeader = Boolean(props.showHeader ?? true);
```

Comment:
Boulder - I tried this widget in the local CMS and noticed when the showHeader value is false, it still renders.

Reply:
note from slack - I haven't reviewed the code, instead, I tested it locally and noticed the showHeader toggle is not working as intended. The header is always rendering regardless of the header value. I believe you added this prop b/c one task did not have a header.  This value should be set by default to true and removed from the CMS.
A header is always required. I think we should ask content what the header is and avoid the cleanup work of removing this variable from all the markdown files once that content bug is fixed.